index.js

```diff
@@ -67,14 +67,16 @@ module.exports = class PreamblePlugin {
         const source = new ConcatSource();
         contents.forEach((entry) => {
           source.add(new OriginalSource(entry.data, entry.source));
+          source.add('\n');
         });
         source.add(src);
+        source.add('\n');
```

Comment:
This shouldn't be necessary.  I don't want to add extra new-lines if they aren't needed.  Please remove it.